### ISSUE TITLE
[rust] next theia-rust images have been failing CI recently

### ISSUE THREAD

**Issue by vince-fugnitto** [theia-ide/theia-apps#253]
**Description**

The next `theia-rust` images have recently been failing during CI (cron) due to the following error:

```
error[E0407]: method `mixed_site` is not a member of trait `server::Span`
   --> /root/rust/cargo/registry/src/github.com-1ecc6299db9ec823/rustc-ap-syntax-606.0.0/ext/proc_macro_server.rs:669:5
    |
669 | /     fn mixed_site(&mut self) -> Self::Span {
670 | |         self.mixed_site
671 | |     }
    | |_____^ not a member of trait `server::Span`
error: aborting due to previous error
For more information about this error, try `rustc --explain E0407`.
error: failed to compile `racer v2.1.28`, intermediate artifacts can be found at `/tmp/cargo-install4fnaaL`
Caused by:
  Could not compile `rustc-ap-syntax`.
To learn more, run the command again with --verbose.
The command '/bin/sh -c . /root/rust/cargo/env && umask 0002 && cargo +$rust_channel install racer' returned a non-zero code: 101
The command "./build_container.sh $NPM_TAG $IMAGE_NAME $NODE_VERSION" failed and exited with 101 during .
```

Only the `theia-rust:next` image has been failing during CI.
cc @dwjbosman 

**Additional Information**

_Last 3 Cron Builds:_
- [dashboard](https://travis-ci.org/theia-ide/theia-apps/builds)
- https://travis-ci.org/theia-ide/theia-apps/jobs/599435330
- https://travis-ci.org/theia-ide/theia-apps/jobs/598951372
- https://travis-ci.org/theia-ide/theia-apps/jobs/597933864

**Comment by dwjbosman** [comment]
I will have a look.
   On Friday, October 18, 2019, 2:33:45 PM GMT+2, Vincent Fugnitto <notifications@github.com> wrote:  
 
 
Description

The next theia-rust images have recently been failing during CI (cron) due to the following error:
error[E0407]: method `mixed_site` is not a member of trait `server::Span`
   --> /root/rust/cargo/registry/src/github.com-1ecc6299db9ec823/rustc-ap-syntax-606.0.0/ext/proc_macro_server.rs:669:5
    |
669 | /     fn mixed_site(&mut self) -> Self::Span {
670 | |         self.mixed_site
671 | |     }
    | |_____^ not a member of trait `server::Span`
error: aborting due to previous error
For more information about this error, try `rustc --explain E0407`.
error: failed to compile `racer v2.1.28`, intermediate artifacts can be found at `/tmp/cargo-install4fnaaL`
Caused by:
  Could not compile `rustc-ap-syntax`.
To learn more, run the command again with --verbose.
The command '/bin/sh -c . /root/rust/cargo/env && umask 0002 && cargo +$rust_channel install racer' returned a non-zero code: 101
The command "./build_container.sh $NPM_TAG $IMAGE_NAME $NODE_VERSION" failed and exited with 101 during .

Only the theia-rust:next image has been failing during CI.
cc @dwjbosman

Additional Information

Last 3 Cron Builds:
   
   - dashboard
   - https://travis-ci.org/theia-ide/theia-apps/jobs/599435330
   - https://travis-ci.org/theia-ide/theia-apps/jobs/598951372
   - https://travis-ci.org/theia-ide/theia-apps/jobs/597933864

**Comment by vince-fugnitto** [comment]
Closed by #254

**Comment by vince-fugnitto** [comment]
The issue still persists.
